raiden/network/proxies/token_network_registry.py

```diff
@@ -228,3 +228,11 @@ def settlement_timeout_min(self) -> int:
     def settlement_timeout_max(self) -> int:
         """ Returns the maximal settlement timeout for the token network registry. """
         return self.proxy.contract.functions.settlement_timeout_max().call()
+
+    def get_token_network_created(self, to_block: BlockSpecification = "latest") -> int:
+        """ Returns the number of TokenNetwork contracts created so far in them
+        token network registry.
+        """
+        return self.proxy.contract.functions.token_network_created().call(
```

Comment:
does `elf.proxy.contract.functions.token_network_created().call()` always return an int? Or can that also throw an error? How is the error case handled?

Reply:
* When the TokenNetworkRegistry is not yet deployed, Web3 throws an exception.
    * Currently that's not handled at all I believe.
* Otherwise, the function always returns an integer.